.changelog/2474.txt

```diff
@@ -0,0 +1,11 @@
+```release-note:enhancement
+resource/mongodbatlas_stream_connection: Supports new `networking` attribute
```

Comment:
```suggestion
resource/mongodbatlas_stream_connection: Adds `networking` attribute
```

nit for all changelog entries

Reply:
changed in 0f965c50b5d71792eb9e730a74600c2078f21b10